wa/workloads/geekbench/__init__.py

```diff
@@ -118,6 +113,7 @@ def __init__(self, *args, **kwargs):
         self.gui.uiauto_params['version'] = self.version
         self.gui.uiauto_params['times'] = self.times
         self.gui.uiauto_params['is_corporate'] = self.is_corporate
+        self.gui.timeout = 3600
```

Comment:
The gui timeout should probably be configurable and related to the `timeout` parameter.